Find a functionally equivalent code:android.widget.CalendarView.getWeekNumberColor:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the color for the week numbers.
 *
 * @return The week number color.
 *
 * @attr ref android.R.styleable#CalendarView_weekNumberColor
 */

@@ -1,3 +1,3 @@
 {
-    return mWeekNumberColor;
+    return mDelegate.getWeekNumberColor();
 }

